### DISCUSSION TITLE
basic-grammar/javadoc

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#270] [Announcements]
# basic-grammar/javadoc

本文详细介绍了Java编程中使用的三种注释方式：单行、多行和文档注释。通过对这三种注释类型的讲解和示例，您将了解它们的应用场景和使用技巧，提高代码的可读性和维护性。

https://javabetter.cn/basic-grammar/javadoc.html

**Comment by AHUA-Official** [comment]
谢谢二哥
